### PR TITLE
Extended readme

### PR DESCRIPTION
**Pull request by pmyagkov**
* Provider extended README for newbies.

### CODE REVIEW

File: config-stub.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+github-user: XXX
+github-password: XXX
+jira-server: https://tangle.atlassian.net
+jira-user: XXX
+jira-password: XXX
+jira-project: FRO
```
Comment (Bahus) [inline config-stub.yml]: Let's add two stub files, one for frontend and seconds for the backend, e.g. `config-stub.backend.yaml`:

```yml
github-user: XXX
github-password: XXX
jira-server: https://tangle.atlassian.net
jira-user: XXX
jira-password: XXX
origin-branch: master
jira-project: BAC
jira-release-project: RELEASE
jira-task-extra:
  component: Backend
release-set: echo
```

Comment (pmyagkov) [reply config-stub.yml]: Fixed.

File: config-stub.backend.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+github-password: XXX
+jira-server: https://tangle.atlassian.net
+jira-user: XXX
+jira-password: XXX
+jira-project: BAC
+jira-release-project: RELEASE
+origin-branch: master
+jira-task-extra:
+  component: Backend
+release-set: yarn version --no-git-tag-version --new-version
```
Comment (Bahus) [inline config-stub.backend.yml]: It's better to set `echo` here, see my comment. Later we are going to change it.

Comment (pmyagkov) [reply config-stub.backend.yml]: Fixed.

**Review by Bahus — APPROVED** [review]

**Pull request merged**